src/Exception/ApiException.php

```diff
@@ -1,8 +1,34 @@
 <?php
 namespace bunq\Exception;
 
+use Exception;
+
 /**
  */
-class ApiException extends BunqException
+class ApiException extends Exception
 {
+    /**
+     * @var int
+     */
+    private $responseCode;
+
+    /**
+     * ApiException constructor.
+     *
+     * @param int $responseCode
+     */
+    public function __construct($message, $responseCode)
```

Comment:
Please add strict types and docs for both parameters!

Reply:
Crap, I’m really getting old indeed xD